post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -6,15 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-[IMPORTANT]
-====
-The Multiarch Tuning Operator is a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
-====
-
 The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration.
 
-This Operator implements the `clusterpodplacementconfigs` custom resource (CR) to support architecture-aware workload scheduling.
+Architecture-aware workload scheduling allows the default {product-title} pod scheduler to place pods onto nodes that match the architecture of the pod's images. 
```

Comment:
```suggestion
Architecture-aware workload scheduling allows the default scheduler to place pods onto nodes that match the architecture of the pod's images. 
```

No pod I think... but also do we need to specify {product-title}? I'm not sure how the kube scheduler is referred in other pages of the doc.

Reply:
@aleskandro 

There are instances of "pod scheduler" and "{product-title} pod scheduler" in the docs:
Link - https://docs.openshift.com/container-platform/4.15/nodes/scheduling/nodes-scheduler-about.html

![Screenshot from 2024-08-29 12-26-11](https://github.com/user-attachments/assets/72646e40-9200-4c8c-8b3f-2eb5967d9f47)